docs/root/version_history/current.rst

```diff
@@ -19,6 +19,7 @@ Bug Fixes
 ---------
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
 
+* cluster: fixed the :ref:`cluster stats <config_cluster_manager_cluster_stats_request_response_sizes>` histograms.
```

Comment:
I think it's worth noting what is being fixed, that we'll now record responses read from upstream even if not delivered downstream, and record per-retry .

Do you think it's worth calling this out in docs as well?  I could imagine it being confusing if the number of upstream entries is higher than downstream, but odds are most folks won't notice and it could be more detail than it's worth :-P

Reply:
Pushing some improvements in a bit to the change log, I am thinking they might be too much detail for the docs? Happy to follow up later on if someone asks :p